CHANGELOG.md

```diff
@@ -0,0 +1,63 @@
+# Changelog
+
+
+## v0.7.0 (TBD)
+
+Starting a Changelog.
+
+### New features and changes
+
+* Dropped Python 2 support. Python >= 3.6 is now required.
+* Added new LicenseListVersion tag.
```

Comment:
This seems comparatively minor to stand at position 2 in the list. Also, it's probably only a part of the "updated the tools"-point

Reply:
I don't think being able to specify the license list version is a 2.2 or 2.3 feature, or is it? But I'll move it further down